react/features/chat/components/web/ChatInput.js

```diff
@@ -113,34 +109,28 @@ class ChatInput extends Component<Props, State> {
                     </div>
                 </div>
                 <div className = 'usrmsg-form'>
-                    <textarea
+                    <TextareaAutosize
                         data-i18n = '[placeholder]chat.messagebox'
                         id = 'usermsg'
+                        inputRef = { this._setTextAreaRef }
+                        maxRows = { 5 }
+                        minRows = { 2 }
```

Comment:
Do you need a minRows prop? I know it's personal preference, I just prefer everything to be as compact as possible until needed to be enlarged.

Reply:
Looking at the designs now it looks like the desire is for 1 line.

Reply:
I changed the initial line to 1. I also "fixed" a UX issue where the input would resize and cover the last message of the chat. I think the impl will need to be redone after other chat changes are merged; I used a setTimeout to get around some timing issues and maybe it won't be needed if the other changes work out.